runtime/vm/FastJNI_java_lang_ref_Reference.cpp

```diff
@@ -51,12 +51,33 @@ Fast_java_lang_ref_Reference_reprocess(J9VMThread *currentThread, j9object_t rec
 	}
 }
 
+/* java.lang.ref.Reference: static private native boolean waitForReferenceProcessingImpl(); */
+jboolean JNICALL
+Fast_java_lang_ref_Reference_waitForReferenceProcessingImpl(J9VMThread *currentThread, j9object_t classObject)
```

Comment:
Is this performance critical?  Does it need to be a fast JNI method?

Reply:
Currently it only is called in nio.Bits.reserveMemory() for java9, but it could be called from 10 times to the maximum as many as the pending reference number in the reference queue if currently the processing references is active.